### ISSUE TITLE
Background color contrast ratio of selected and hovering dropdown item is less than 3:1

### ISSUE THREAD

**Issue by iwasn** [OfficeDev/office-ui-fabric-react#10464]
### Environment Information

- **Package version(s)**: 7.34.0 (Current latest)
- **Browser and OS versions**: Any

### Describe the issue:

Non text contrast ratio of both the selected and hovered options(grey vs white background) are less than the minimum 3:1 requirement.

### Please provide a reproduction of the issue in a codepen:

Example from demo site: 
https://fabricweb.z5.web.core.windows.net/oufr/7.34.0/#/examples/dropdown

![image](https://user-images.githubusercontent.com/54422491/64995930-6356c800-d891-11e9-8078-20a201de5742.png)

#### Actual behavior:

Color contrast should meet accessibility threshold.

#### Expected behavior:

Contrast does not meet accessibility threshold.

### Documentation describing expected behavior

w3 accessibility criteria: https://www.w3.org/TR/WCAG21/#non-text-contrast.

**Comment by dzearing** [comment]
@betrue-final-final Thoughts?

**Comment by betrue-final-final** [comment]
Hover has no requirement but selected does if there is no other indicator of selection. I suspect that if the value is in the dropdown then it is ok.

**Comment by micahgodbolt** [comment]
waiting on design to see if we need to make a change to our dropdown.

**Comment by micahgodbolt** [comment]
per @betrue-final-final 

> Dropdown actually does not have an issue as hover is not included in the non-text rule, and selection is also indicated in the dropdown itself, so you are actually safe.

> There is no requirement for pressed and hover, especially as the menu will dismiss and read the new value in the dropdown.
